### PR TITLE
add skull emoji reaction option

### PR DESCRIPTION
**Pull request by 24c02**
i'm mostly kidding but on the off chance that y'all would be crazy enough to merge this:

### CODE REVIEW

File: app/models/comment/reaction.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ class Reaction < ApplicationRecord
     validates :emoji, presence: true
     validates :emoji, uniqueness: { scope: [:reactor_id, :comment_id], message: "has already been used for this comment" }
 
-    EMOJIS = %w[👍 👎 😄 🎉 😔 ❤️ 🚀 👀].freeze
+    EMOJIS = %w[👍 👎 😄 🎉 😔 ❤️ 🚀 👀 💀].freeze
```
Comment (garyhtou) [inline app/models/comment/reaction.rb]: half the deploys have a 💀 /j

```suggestion
    EMOJIS = (%w[👍 👎 😄 🎉 😔 ❤️ 🚀 👀] + (rand > 0.5 ? ["💀"] : [])).freeze
```

Comment (24c02) [reply app/models/comment/reaction.rb]: Screenshot 2024-08-29 at 10 38 04 PM

github refuses to let me respond appropriately

Comment (garyhtou) [reply app/models/comment/reaction.rb]: LOL

**Review by sampoder — APPROVED** [review]
lol let's do it... at least for a bit and see if it's used

**Pull request merged**